src/rpc.ts

```diff
@@ -20,7 +26,9 @@ export interface IRequestStream<T> {
   end(): void;
 }
 
-export interface IDuplexStream<T, R> extends IRequestStream<T>, IResponseStream<R> {}
+export interface IDuplexStream<T, R> extends IRequestStream<T>, IResponseStream<
+  R
```

Comment:
yea, I agree. I can change prettier's wrapping to 100 columns instead of 80 which would reduce these scenarios